packages/table-react/src/TableRowAnchor.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { TableRowAnchorData } from "./Table";
+import { TableData } from "./TableData";
+
+interface PropTypes {
+    row: TableRowAnchorData;
+}
+
+export function TableRowAnchor({ row }: PropTypes) {
+
+    const onClick = () => {
+        if (row.onRowClick) {
+            row.onRowClick(row.href);
+        } else {
+            window.location.href = row.href;
```

Comment:
I always get a bit anxious when i see window.stuff in libary code. At least we should check if window is defined before firing off this. And in general modifying the history object is a can of worms.